# Python NameError: name 'self' is not defined Why?
python -tt
